tests/Tests/500__jupyterhub/culler.robot

```diff
@@ -0,0 +1,169 @@
+*** Settings ***
+Documentation    Tests for the NB culler
+Resource         ../../Resources/ODS.robot
+Resource         ../../Resources/Common.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource         ../../Resources/Page/ODH/JupyterHub/JupyterLabLauncher.robot
+Resource         ../../Resources/Page/ODH/ODHDashboard/ODHDashboard.robot
+Library          ../../../libs/Helpers.py
+Library          OpenShiftLibrary
+Suite Teardown   End Web Test
+Force Tags       JupyterHub
+
+
+*** Variables ***
+${DEFAULT_CULLER_TIMEOUT} =    31536000
+${CUSTOM_CULLER_TIMEOUT} =     300
+
+
+*** Test Cases ***
+Verify Default Culler Timeout
+    [Documentation]    Checks default culler timeout
+    [Tags]  Sanity    Tier1
+    ...    ODS-1255
+    Disable Culler
+    ${current_timeout} =  Get Culler Timeout
+    Should Be Equal  ${DEFAULT_CULLER_TIMEOUT}  ${current_timeout}
+    Close Browser
+
+Verify Culler Timeout Can Be Updated
+    [Documentation]    Verifies culler timeout can be updated
+    [Tags]  Sanity    Tier1
+    ...    ODS-1231
+    Modify Culler Timeout    ${CUSTOM_CULLER_TIMEOUT}
+    ${current_timeout} =  Get Culler Timeout
+    Should Not Be Equal  ${current_timeout}  ${DEFAULT_CULLER_TIMEOUT}
+    Should Be Equal   ${current_timeout}  ${CUSTOM_CULLER_TIMEOUT}
+    Close Browser
+
+Verify Culler Kills Inactive Server
+    [Documentation]    Verifies that the culler kills an inactive 
+    ...    server after timeout has passed.
+    [Tags]    Sanity    Tier1
+    ...    ODS-1254
+    Spawn Minimal Image
+    Clone Git Repository And Open    https://github.com/redhat-rhods-qe/ods-ci-notebooks-main    notebooks/500__jupyterhub/Inactive.ipynb
```

Comment:
I think it will be easier to manage in the future if you create a culler or notebook-culler subfolder in the repo:
`notebooks/500__jupyterhub/culler/Inactive.ipynb`

Reply:
Makes sense, will do.

Reply:
Done